doc/lua_api.txt

```diff
@@ -4841,6 +4841,8 @@ Definition tables
     --  ^ "wielditem" expects 'textures = {itemname}' (see 'visual' above).
         colors = {},
     --  ^ Number of required colors depends on visual.
+        use_texture_alpha = false,
+    --  ^ Use texture's alpha channel, excludes "upright_sprite" and "wielditem"
```

Comment:
should maybe add something like:

    -- ^ Note: currently causes visual issues when viewed through a semi-transparent material such as water

Reply:
I am happy to add that, however, this is also true for translucent nodes so perhaps a more general note about the use alpha channel would be appropriate?

Regarding a fix, I think the best we can ever hope to do is improve things, even big commercial games struggle with this problem. The general advice is to find the best compromise for your game and avoid using translucency wherever possible, which is particularly difficult for a sandbox game/engine like MT.

Reply:
very true